functions/release_notes/main.py

```diff
@@ -0,0 +1,45 @@
+import logging
+import os
+
+import flask
+import functions_framework
+
+from bugbug import generative_model_tool
+from bugbug.tools.release_notes import ReleaseNotesCommitsSelector
+from bugbug.utils import get_secret
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+os.environ["OPENAI_API_KEY"] = get_secret("OPENAI_API_KEY")
```

Comment:
When using `create_openai_llm()`, it will read the secret directly. Why do we need this?